torch/_linalg_utils.py

```diff
@@ -96,34 +96,69 @@ def symeig(A: Tensor, largest: Optional[bool] = False) -> Tuple[Tensor, Tensor]:
 def matrix_rank(input, tol=None, symmetric=False, *, out=None) -> Tensor:
     raise RuntimeError(
         "This function was deprecated since version 1.9 and is now removed.",
-        "Please use the `torch.linalg.matrix_rank` function instead.",
+        "Please use the `torch.linalg.matrix_rank` function instead,",
+        "The parameter 'symmetric' was renamed in `torch.linalg.matrix_rank()` to 'hermitian'.",
```

Comment:
In fact, I believe that what you want is to remove all the commas. Can you double check how does the formatting look? I think that with the commas you're raising a tuple. Without the commas, all the strings get concat together.

Reply:
Also, you'll need spacing at the end of the sentences and so on. Mind testing how all these look locally?